our-brothers-web/src/app/shared/components/view-toggle/view-toggle.component.html

```diff
@@ -1,14 +1,30 @@
 <div class="view-toggle-container">
```

Comment:
There is a duplication of code here. Please add a TODO comment with the message: 
``` html
<!-- TODO: Angular templates optimizations -->
```

Reply:
fix